src/main/java/frc/robot/hardware/rev/motors/SparkMaxMotor.java

```diff
@@ -65,8 +65,10 @@ public void updateInputs(InputSignal<?>... inputSignals) {
 
 	@Override
 	public void setBrake(boolean brake) {
-		CANSparkBase.IdleMode idleMode = brake ? CANSparkBase.IdleMode.kBrake : CANSparkBase.IdleMode.kCoast;
-		motor.setIdleMode(idleMode);
+		SparkBaseConfig.IdleMode idleMode = brake ? SparkBaseConfig.IdleMode.kBrake : SparkBaseConfig.IdleMode.kCoast;
+
+		//FIXME motor.configure()
+//		motor.setIdleMode(idleMode);
```

Comment:
What's the problem with this?

Reply:
This doesnt exist anymore, rev changed the whole way of configuring their devices

More than welcome to look here https://github.com/REVrobotics/REV-Software-Binaries/releases/tag/revlib-2025.0.0-beta-1

![תמונה של WhatsApp‏ 2024-12-26 בשעה 19 32 10_f129e4ea](https://github.com/user-attachments/assets/5e613dd9-73ff-412a-ae9b-5cb08c492de2)